### PR TITLE
non default db connection bug patch

### PR DESCRIPTION
**Pull request by ashexpertVersion2**

### CODE REVIEW

File: health_check/mixins.py [hunk 1]
```diff
@@ -33,8 +33,9 @@ def _run(plugin):
             try:
                 return plugin
             finally:
-                from django.db import connection
-                connection.close()
+                from django.db import connections
+                for connection in connections.all():
+                    connection.close()
```
Comment (codingjoe) [inline health_check/mixins.py]: There is actually a method on the `ConnectionHandler` that is very similar, but also add some exception handling for already closed connections.
```suggestion
                from django.db import connections
                connections.close_all()
```

File: tests/testapp/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
-from .celery import app
+from .celery_app import app
```
Comment (codingjoe) [inline tests/testapp/__init__.py]: This change is not related and also not needed. Please revert the change.
```suggestion
from .celery import app
```
You will also need to rename the file again.

File: tests/testapp/settings.py [hunk 3]
```diff
@@ -9,6 +9,10 @@
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': ':memory:',
+    },
+    'backup': {
```
Comment (codingjoe) [inline tests/testapp/settings.py]: IMHO, we might want to give it a more generic name and a comment, to not confuse people.
```suggestion
    'other': {  # 2nd database conneciton to ensure proper connection handling
```

**Review by codingjoe — COMMENTED** [review]
Hi @ashexpertVersion2 thanks for the contribution. You are off to a very good start. I did add some comments that might help to improve your patch. Please let me know if you have any questions.
Best -Joe

File: tests/testapp/__init__.py (hunk 2 above)
Comment (ashexpertVersion2) [reply tests/testapp/__init__.py]: i couldn't run tests due to circular dependency caused by similarity of this file's name and the celery module. you are right i shouldn't add it to this commit but it slipped out of my hand.

**Review by codingjoe — APPROVED** [review]
LGTM Thanks again @ashexpertVersion2 much appreciated! 👍

**Comment by codingjoe** [comment]
Released in [3.12.2](https://github.com/KristianOellegaard/django-health-check/releases/tag/3.12.2) 🎉

**Comment by ashexpertVersion2** [comment]
Your welcome :+1: 
thank you for fast response.
